services/QuillLMS/app/controllers/blog_posts_controller.rb

```diff
@@ -24,34 +24,18 @@ def student_center_index
     render :index
   end
 
-  # rubocop:disable Metrics/CyclomaticComplexity
   def show
```

Comment:
Nice job refactoring this and lowering the cyclomatic complexity.  The code is much easier to read now.